app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -214,8 +229,11 @@ function ProjectFramework(props: Props) {
         offset,
     ]);
 
-    const totalFrameworksCount = frameworksResponse?.count ?? 0;
+    const totalFrameworksCount = projectUsersResponse?.analysisFrameworks?.totalCount ?? 0;
+    const AfList: FrameworkMini[] = projectUsersResponse?.analysisFrameworks?.results || [];
```

Comment:
```suggestion
    const frameworkList: FrameworkMini[] = projectUsersResponse?.analysisFrameworks?.results || [];
```
Ts variables should always be camelcase.